build/InstallAndroid.ps1

```diff
@@ -0,0 +1,33 @@
+$url = "https://dl.google.com/android/repository/commandlinetools-win-7583922_latest.zip"
+$fileName = "AndroidTools.zip"
+$source = "C:\Downloads\$fileName"
+$destination = "C:\Downloads\AndroidSdkTools"
+$androidSdk = "C:\Program Files (x86)\Android\android-sdk\"
+
+#$ErrorActionPreference = "Stop"
+
+if (Test-Path "$PSScriptRoot\win-installer-helper.psm1") 
+{
+    Import-Module "$PSScriptRoot\win-installer-helper.psm1" -DisableNameChecking
+} elseif (Test-Path "$PSScriptRoot\..\..\Helpers\win-installer-helper.psm1") 
+{
+    Import-Module "$PSScriptRoot\..\..\Helpers\win-installer-helper.psm1" -DisableNameChecking
+}
+
+mkdir -Path "C:\Program Files (x86)\Android\android-sdk\licenses" -Force
+
+Get-File -Url $url -FileName $fileName
+
+echo "Expanding"
+Expand-Archive -LiteralPath "$source" -DestinationPath "C:\Downloads\AndroidSdkTools" -Force
+
+dir "C:\Program Files (x86)\Android\android-sdk\licenses"
+
+echo "Installing licenses"
```

Comment:
nit: maybe add a comment here, that this is a workaround, so we remember to update this, if OneBranch fixes this?